### DISCUSSION TITLE
Distributing command sequences throughout CmdSequencers

### DISCUSSION THREAD

**Discussion by zimri-leisher** [nasa/fprime#2608] [Q&A] [answered]
Hello,
We'd like to run multiple command sequences at the same time. As I understand, that means we need more than one command sequencer. However, it would be nice not to have to decide which command sequencer we want to run a command on. Instead, we want to just say "run this sequence on an available cmd sequencer".
Any suggestions for this? It seems like it would be a common problem. Or is this a bad idea and we should have to explicitly declare which command sequencer we want to use?

**Comment by timcanham** [comment, marked as answer]
@zimri-leisher It's actually in our roadmap... We have been discussing a component that would act as a front end for a set of sequencers that would dispatch the sequence to the next available instance.

**Reply by zimri-leisher** [reply]
Ok well this sounds like another thing I might submit a PR for. Will let you know how it goes :)
